agr_java_core/src/main/java/org/alliancegenome/neo4j/repository/GeneRepository.java

```diff
@@ -533,23 +530,48 @@ private String getAllGenesQuery(OrthologyFilter filter) {
         return stageList;
     }
 
-    public Map<String, String> getFullAoList() {
-        String cypher = "match p=(uber:UBERONTerm)-[:ANATOMICAL_RIBBON_TERM]-(:ExpressionBioEntity) return distinct uber";
+    private LinkedHashMap<String, String> aoOrderList;
+    private Map<String, Integer> aoOrderedPositionList;
 
-        Iterable<UBERONTerm> terms = query(UBERONTerm.class, cypher);
-        String alwaysLast = "other";
-        Map<String, String> map = StreamSupport.stream(terms.spliterator(), false)
-                .sorted((o1, o2) -> {
-                    if (o1.getName().equalsIgnoreCase(alwaysLast)) {
-                        return 1;
-                    }
-                    if (o2.getName().equalsIgnoreCase(alwaysLast)) {
-                        return -1;
-                    }
-                    return o1.getName().compareTo(o2.getName());
-                })
-                .collect(Collectors.toMap(UBERONTerm::getPrimaryKey, UBERONTerm::getName, (x, y) -> x + ", " + y, LinkedHashMap::new));
-        return map;
+    public LinkedHashMap<String, String> getOrderAoTermList() {
+        if (aoOrderList != null)
+            return aoOrderList;
+        LinkedHashMap<String, String> aoTermList = new LinkedHashMap<>();
+        InputStream in = null;
+        BufferedReader reader = null;
+        try {
+            String str = null;
+            in = this.getClass().getClassLoader().getResourceAsStream("anatomy-term-order.csv");
+            if (in != null) {
+                reader = new BufferedReader(new InputStreamReader(in));
+                while ((str = reader.readLine()) != null) {
+                    String[] token = str.split("\t");
+                    aoTermList.put(token[0], token[1]);
+                }
+            }
```

Comment:
@cmpich can we put this code into the ConfigHelper file... we might need to do more of this kind of thing in the future. Something like `aoTermList = ConfigHelper.getAnatomyTermOrderList()` that way the paths can be handled there also.

Reply:
Also can we keep ALL class variables at the top of the class... so that way they are all in one place and don't have to go hunting to find them?

Reply:
Yes. I will need it for the GO list as well.